layouts/archive/single.html

```diff
@@ -0,0 +1,70 @@
+{{ define "main" }}
+	<div class="row flex-xl-nowrap">
+
+		<div class="col-lg-5 col-xl-4 docs-sidebar{{ if ne site.Params.doks.navbarSticky true }} docs-sidebar-top{{ end }}{{ if site.Params.doks.headerBar }} docs-sidebar-offset{{ end }} d-none d-lg-block">
+			{{ partial "sidebar/section-menu.html" . }}
+		</div>
+
+		{{ if and (eq site.Params.doks.containerBreakpoint "fluid") (in .Site.Params.mainSections .Type) }}
+			<div class="col container-fw d-lg-flex flex-lg-row justify-content-center mx-auto">
+		{{ end }}
+		{{ if ne .Params.toc false -}}
+		<nav class="docs-toc{{ if ne site.Params.doks.navbarSticky true }} docs-toc-top{{ end }}{{ if site.Params.doks.headerBar }} docs-toc-offset{{ end }} d-none d-xl-block col-xl-3" aria-label="Secondary navigation">
+			{{ partial "sidebar/docs-toc-desktop.html" . }}
+		</nav>
+		{{ end -}}
+		{{ if .Params.toc -}}
+		<main class="docs-content col-lg-11 col-xl-9">
+		{{ else -}}
+		<main class="docs-content col-lg-11 col-xl-9 mx-xl-auto">
+		{{ end -}}
+			{{ if site.Params.doks.breadcrumbTrail -}}
+				<!-- https://discourse.gohugo.io/t/breadcrumb-navigation-for-highly-nested-content/27359/6 -->
+				<nav aria-label="breadcrumb">
+					<ol class="breadcrumb">
+						{{ partial "main/breadcrumb" . -}}
+						<li class="breadcrumb-item active" aria-current="page">{{ .Title }}</li>
+					</ol>
+				</nav>
+			{{ end }}
+			
+			<h1>{{ .Title }}</h1>
+
+			<div class="grey-box">
+                <span class="alert-icon">&#9888;</span> This documentation refers to a version of Oakestra which is not the latest. Please refer to <a href="/docs">latest</a>.
```

Comment:
```suggestion
                <span class="alert-icon">&#9888;</span> This page documents a previous version of Oakestra. Please refer to the <a href="/docs/getting-started/oak-environment/high-level-setup-overview/">latest docs</a> for a current version.
```
I think the phrasing here is a bit off